front/app/modules/commercial/content_builder/admin/components/CraftComponents/AboutBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+// components
+import ProjectInfoSideBar from '../../../../../../../../../front/app/containers/ProjectsShowPage/shared/header/ProjectInfoSideBar';
+import { Box } from '@citizenlab/cl2-component-library';
```

Comment:
This import statement is a little long, but is this fine? Or should the component be duplicated in the module?

Reply:
Can you do an absolute import here, and not a relative one? import ProjectInfoSideBar from `containers/ProjectsShowPage/shared/header/ProjectInfoSideBar`